frontend/packages/integration-tests-cypress/support/a11y.ts

```diff
@@ -66,6 +66,7 @@ Cypress.Commands.add('testA11y', (target: string, selector?: string) => {
       { id: 'color-contrast', enabled: false }, // seem to be somewhat inaccurate and has difficulty always picking up the correct colors, tons of open issues for it on axe-core
       { id: 'focusable-content', enabled: false }, // recently updated and need to give the PF team time to fix issues before enabling
       { id: 'scrollable-region-focusable', enabled: false }, // recently updated and need to give the PF team time to fix issues before enabling
+      { id: 'aria-roles', enabled: false }, // rule is outdated and current installation of axe-core is no longer complaint with newest ariaRoles spec, causing false test failures (specifically with the role='code'). reenable when axe core is updated to at least 4.1.0
```

Comment:
note that an accessibility check is disabled due to an outdated implementation in our current version of `axe-core`, upgrading in turn is blocked by various accessibility issues in console that must be resolved